x-pack/plugins/apm/server/plugin.ts

```diff
@@ -47,14 +59,25 @@ export class APMPlugin implements Plugin<APMPluginContract> {
       licensing: LicensingPluginSetup;
       cloud?: CloudSetup;
       usageCollection?: UsageCollectionSetup;
+      taskManager?: TaskManagerSetupContract;
     }
   ) {
-    const logger = this.initContext.logger.get('apm');
+    const logger = this.initContext.logger.get();
```

Comment:
Why no longer in the context of 'apm'?

Reply:
it's already scoped, so scoping it again leads to `[apm] [apm]` output